Expecto.Differ/Library.fs

```diff
@@ -0,0 +1,45 @@
+module Expecto.Differ
+open System
+open DiffPlex.DiffBuilder
+open DiffPlex.DiffBuilder.Model
+
+let colorizedDiff (colorizeText: ConsoleColor -> string -> string) first second =
+  let first, second =
+    match box first, box second with
+    | (:? string as f), (:? string as s) ->
+      string f, string s
+    | f, s ->
+      sprintf "%A" f, sprintf "%A" s
```

Comment:
This will generate different indentation depending on the values of the fields in the objects. Of course, this is still an improvement, but a tree-walking algo that does consistent indentation would be preferred.